docs/app/Examples/elements/Label/Variations/LabelExampleSize.js

```diff
@@ -0,0 +1,19 @@
+import _ from 'lodash'
+import React from 'react'
+import { Label } from 'stardust'
+
+const LabelExampleSize = () => {
+  const sizes = ['mini', 'tiny', 'small', 'medium', 'large', 'big', 'huge', 'massive']
```

Comment:
Actually, I think better would be to just use `Label._meta.props.size`, this way the sizes we give examples for are all the sizes the component supports. There are a few other places in the docs we should update to do this as well.